app/scripts/controllers/network/network-controller.js

```diff
@@ -125,7 +125,9 @@ export default class NetworkController extends EventEmitter {
     }
     this._infuraProjectId = infuraProjectId;
 
-    this.on(NETWORK_EVENTS.NETWORK_DID_CHANGE, this.lookupNetwork);
+    this.on(NETWORK_EVENTS.NETWORK_DID_CHANGE, () => {
```

Comment:
Revisiting this again, it turns out that in order to effectively stub out this method using `jest.spyOn` in the tests I have to wrap the method call like this.

Reply:
Context for any future readers: we discovered that this change is a no-op because of how the event emitter works: https://github.com/MetaMask/metamask-extension/pull/17498/files#r1092527202